eng/scripts/New-TestResources-Bootstrapper.ps1

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is this not in Azure/azure-sdk-tools?

Reply:
This is only going to be used by the .NET test suite.